docs/resources/datastore.md

```diff
@@ -151,13 +151,17 @@ Required:
 ## Example Usage
 
 ```terraform
-locals {
-  dataaccess_controller_id = "<assigned dataaccess_controller_id>"
+data "satori_data_access_controller" "dac0" {
```

Comment:
let's keep both examples

Reply:
for example: datastore0 will use id and datastoreWithIgnorePasswordUpdate will use the new object.

If you have id there is no need to request the dac object